src/sb_pointcloud_processing/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     set_property(CACHE CMAKE_BUILD_TYPE PROPERTY VALUE "Release")
 endif()
 
-add_definitions(-std=c++14)
+add_definitions(-std=c++14 -fopenmp)
```

Comment:
I'm not sure what happens if we have a compiler flag for a feature that isn't present on the computer. A better way to add OpenMP might be the following (which would go below with the rest of the `find_package` lines):
```
find_package(OpenMP REQUIRED)
if(OPENMP_FOUND)
message("OPENMP FOUND")
    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
endif()
```
cmake doc reference: https://cmake.org/cmake/help/v3.0/module/FindOpenMP.html

I think this would also mean you wouldn't have to link against `gomp` for all the executables (the compiler would just have universal openmp support). Normally linking something to every executable is bad practice, but in this case the compiler is literally just replacing your code with a multithreaded version before it compiles it (the actual multithreading is done with `pthread`, so you *might* have to link against that), so there's no performance loss.

Reply:
done